content/md/assets/about-product-link-rules.md

```diff
@@ -38,6 +39,20 @@ The naming convention can be defined via the API or directly in the PIM UI, in t
 
 ![Assets Naming Convention](../img/Assets_NamingConventionNew.png)
 
+### Should I split the asset code or the filename?
+
+As said above, you can choose to split the asset code or the main media filename. What should you do? Let's discover it.
+
+#### First use case
+If you **mass upload** your assets in the Asset Manager - meaning that your main media attribute is a `media file` attribute - you can choose to **split either the code of the asset, or its filename**.   
+
+#### Second use case
+If you create your assets **one by one** in your family, or if the main media attribute of your asset family is a `media link` attribute (meaning that you **can't use the mass upload** feature), you must split the **code** of your assets, **not the filename**.
+
+*Why?*
+Because the naming convention is executed at the ***asset creation*** step. In the second use case, we don't know the filename of the asset at the asset creation since we add the file afterward. That's why the naming convention won't work if you split the filename instead of the code :wink:
```

Comment:
```suggestion
Because the naming convention is executed at the ***asset creation*** step. In the second use case, we don't know the filename of the asset when the asset creation takes place since we add the file afterward. That's why the naming convention won't work if you split the filename instead of the code :wink:
```